### DISCUSSION TITLE
PayAccount 에서 유저를 구분하는 방법 논의

### DISCUSSION THREAD

**Discussion by kimhyun5u** [woowa-techcamp-2024/Team1-3K1K#36] [General]
논의 배경: PayAccount 에서 Customer 와 Vendor 를 구분할 수 있는 방법이 존재하지 않는다.
최악의 경우 member_id 하나에 2개의 PayAccount 가 생길 수 있다.

**Comment by Hyeon-Uk** [comment]
<img width="625" alt="image" src="https://github.com/user-attachments/assets/d75bbddf-2d3e-4926-b706-20ffd50870fa">


문제에 대한 간단한 이미지를 첨부합니다!

**Comment by kimhyun5u** [comment]
추가적 논의 배경: Customer 와 Vendor Login 을 하나로 합쳐야하나? -> 상위 테이블이 있어야겠다. -> 계좌 이렇게 해야한다. -> 송금 시 계좌를 조회하기 위헤서 2 테이블을 모두 조회해야함 (최대 4번 조회 )-> 기록 어떻게 남기냐

**Comment by Hyeon-Uk** [comment]
결론 : Vendor와 Customer의 PK를 UUID로 하여 중복값이 발생하지 않도록 한다.
